src/FS.Embedded.Manifest.Task/GenerateEmbeddedResourcesManifest.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+
+namespace Microsoft.Extensions.FileProviders.Embedded.Manifest.Task
+{
+    /// <summary>
+    /// Task for generating a manifest file out of the embedded resources in an
+    /// assembly.
+    /// </summary>
+    public class GenerateEmbeddedResourcesManifest : Microsoft.Build.Utilities.Task
+    {
+        private const string LogicalName = "LogicalName";
+        private const string ManifestResourceName = "ManifestResourceName";
+        private const string TargetPath = "TargetPath";
+
+        [Required]
+        public ITaskItem[] EmbeddedFiles { get; set; }
+
+        [Required]
+        public ITaskItem ManifestFile { get; set; }
```

Comment:
Seems like the only usage of this is to acquire the `TargetPath` metadata item, right? Any reason this couldn't just be `public string ManifestFilePath { get; set; } `?